modules/migration-mtc-support.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="migration-mtc-support_{context}"]
```

Comment:
```suggestion
:_mod-docs-content-type: CONCEPT
[id="migration-mtc-support_{context}"]
```

Reply:
I think this should be a reference module. Tabular format would be more scannable and readable.

Reply:
There is always the question of accessibility with tables (in the context of screen readers)as this data is simple i think it is probably best to use bullet point